gst/nnstreamer/tensor_decoder/tensordec.c

```diff
@@ -675,7 +694,7 @@ gst_tensordec_transform (GstBaseTransform * trans,
   if (G_UNLIKELY (!self->configured))
     goto unknown_format;
 
-  if (self->decoder) {
+  if (self->decoder || self->is_custom) {
```

Comment:
Code clean: do we need this if? checks at :648 makes this if always true.

Reply:
Well.. The two conditions are not directly related.